### ISSUE TITLE
Broken test for rest API

### ISSUE THREAD

**Issue by remicollet** [glpi-project/glpi#1057]
```
$ phpunit

GLPI dataset version 2 already loaded

PHPUnit 5.5.5 by Sebastian Bergmann and contributors.

.FFSSSSSSSSSSSSSSSSSSSSSFFSSSSSSSSSSSSSSSSSSSS.................  63 / 155 ( 40%)
............................................................... 126 / 155 ( 81%)
............................

Time: 2.88 seconds, Memory: 58.00MB

There were 4 failures:

1) APIRestTest::testInitSessionCredentials
HTTP/1.1 400 Bad Request
Host: localhost:8088
Connection: close
X-Powered-By: PHP/5.6.26
Pragma: no-cache
Access-Control-Expose-Headers: content-type, content-range, accept-range
Content-Type: application/json; charset=UTF-8
Cache-Control: no-store, no-cache, must-revalidate
Expires: Mon, 26 Jul 1997 05:00:00 GMT

["ERROR_APP_TOKEN_PARAMETERS_MISSING","le paramètre app_token est manquant; Afficher la documentation dans votre navigateur à http://localhost/glpimaster/api/#ERROR_APP_TOKEN_PARAMETERS_MISSING"]
Failed asserting that null is not equal to null.

/work/GLPI/master/tests/API/APIRestTest.php:86

2) APIRestTest::testInitSessionUserToken
HTTP/1.1 400 Bad Request
Host: localhost:8088
Connection: close
X-Powered-By: PHP/5.6.26
Pragma: no-cache
Access-Control-Expose-Headers: content-type, content-range, accept-range
Content-Type: application/json; charset=UTF-8
Cache-Control: no-store, no-cache, must-revalidate
Expires: Mon, 26 Jul 1997 05:00:00 GMT

["ERROR_APP_TOKEN_PARAMETERS_MISSING","le paramètre app_token est manquant; Afficher la documentation dans votre navigateur à http://localhost/glpimaster/api/#ERROR_APP_TOKEN_PARAMETERS_MISSING"]
Failed asserting that null is not equal to null.

/work/GLPI/master/tests/API/APIRestTest.php:112

3) APIXmlrpcTest::testInitSessionCredentials
Failed asserting that null is not equal to false.

/work/GLPI/master/tests/API/APIXmlrpcTest.php:59

4) APIXmlrpcTest::testInitSessionUserToken
Failed asserting that null is not equal to false.

/work/GLPI/master/tests/API/APIXmlrpcTest.php:79

FAILURES!
Tests: 114, Assertions: 1382, Failures: 4, Skipped: 41.
```

Notice, is config/config_db.php is same than test/config_db.php everything works, but this is obviously not correct.

Test instance should not rely on Dev instance, both are separate by design.

**Comment by remicollet** [comment]
Additional, it would be nice to have url_base define as a constant in phpunit.xml and used in bootstrap.

**Comment by btry** [comment]
Hi @remicollet 

I would also appreciate to have some cosntants in phpunit.xml. What do you think about the abobe mentionned commit in my fork ?

**Comment by remicollet** [comment]
@btry too late ;)  already fixed in #1058
